runhouse/servers/cluster_servlet.py

```diff
@@ -46,6 +47,10 @@ async def __init__(
         if not cluster_config:
             cluster_config = load_cluster_config_from_file()
 
+        if cluster_config:
```

Comment:
maybe move this a couple lines lower until after we've already set `self.cluster_config`? that way we could also remove this check